docker/jenkins/Dockerfile.snyk

```diff
@@ -0,0 +1,10 @@
+ARG BASE_IMAGE=ubuntu:bionic
+FROM 263245908434.dkr.ecr.us-east-1.amazonaws.com/$BASE_IMAGE
```

Comment:
Is there a reason we need to use our actual build image here instead of just a lighter weight base image?

Reply:
I don't believe I changed this from the one in the pro repo. We could absolutely change it to something lighter weight, but I haven't looked into that.

Reply:
Ahh I didn't realize this already existed in the Pro repo.